packages/react-dom/src/client/ReactDOMComponent.js

```diff
@@ -206,6 +207,26 @@ if (__DEV__) {
     testElement.innerHTML = html;
     return testElement.innerHTML;
   };
+
+  formatTagDEV = function(node: Element): string {
+    let str = '<' + node.nodeName.toLowerCase();
+    const attributeNames = node.getAttributeNames();
+    for (let i = 0; i < attributeNames.length; i++) {
+      if (i > 30) {
+        str += ' ...';
+        break;
+      }
+      const attributeName = attributeNames[i];
```

Comment:
I could get a React name for this but seems confusing. E.g. it would still print `style` as a string. So I figured it should just use HTML in the message.